pytest_html/plugin.py

```diff
@@ -440,6 +440,8 @@ def _generate_environment(self, config):
             value = metadata[key]
             if isinstance(value, basestring) and value.startswith('http'):
                 value = html.a(value, href=value, target='_blank')
+            elif isinstance(value, (list, tuple, set)):
+                value = ', '.join(value)
```

Comment:
This fails with `TypeError: sequence item 0: expected str instance, int found` if `value` is a collection of integers.

Reply:
Fixed by converting all items in collection to strings before join.